test/kubelessDeploy.test.js

```diff
@@ -61,6 +60,9 @@ function instantiateKubelessDeploy(handlerFile, depsFile, serverlessWithFunction
 
 function mockThirdPartyResources(kubelessDeploy) {
   const thirdPartyResources = {
+    namespaces: {
+      namespace: 'default',
```

Comment:
we will need to check that we can deploy a function in a different namespace than the default. i am not sure what serverless schema key we can use.